client-code/Dockerfile

```diff
@@ -6,6 +6,9 @@ COPY client-code/ .
 COPY interface-code/ .
 RUN chmod +x entrypoint.sh
 
-RUN javac *.java
+RUN mkdir target/
```

Comment:
shoudl be created as a static dir inside the repo with `.gitkeep` inside and listed in `.gitignore` and `.dockerignore`.